WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -3234,6 +3240,9 @@ static GSourceFuncs _handlerIntervention =
             // webaudio support
             WKPreferencesSetWebAudioEnabled(preferences, _config.WebAudioEnabled.Value());
 
+            // Turn on/off ICE candidate filtering
```

Comment:
Do you really use wpe-2.22 still? WebRTC is quite old there, it probably just drop the change.

Reply:
We don't use 2.22 but I put it there just for consistency. It doesn't harm in case value is not set at all.

Reply:
Since there is no use case (and an easy way to build it in sprint) I'd prefer to not add it. Could you please remove?